### PR TITLE
SC-2071 resolved indentation changes

### PR DESCRIPTION
**Pull request by Hari-stackroute**

### CODE REVIEW

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://656.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br />  <br/> **Modified sections are -** <br />

File: apis/definitions/schemas/address.yaml [hunk 1]
```diff
@@ -22,16 +24,12 @@ definitions:
...
-	 format: yyyy-MM-dd HH:mm:ss:SSSZZZZ
-      isDeleted:
-          type: boolean
-      createdBy:
-          type: string
-      userId:
-          type: string	  
-     
-     
+  	    type: string
```
Comment (SMYALTAMASH) [inline apis/definitions/schemas/address.yaml:30]: please validate all files before raising PR, this indentation is not valid.

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://656.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br />  <br/> **Modified sections are -** <br />

File: apis/definitions/schemas/address.yaml (hunk 1 above)
Comment (Hari-stackroute) [reply apis/definitions/schemas/address.yaml]: @SMYALTAMASH now it is proper. please check.

**Comment by SMYALTAMASH** [comment]
PR Looks good and can be merged.

**Comment by keshavprasadms** [comment]
Hello @Hari-stackroute,<br/><br/> _This is a auto-generated response._ <br/><br/> Click the URL for review pull request - http://656.qa.docs.sunbird.org/api-documentation/ .<br/> **Note:** Only _api-documentation_ version was generated for PR preview. <br/><br/> **Modified pages are -** <br />- http://656.qa.docs.sunbird.org/api-documentation/apis/consentapi// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/datasyncapi// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/definitions/schemas// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/firebasecloudmessagingapi// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/geolocationapi// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/locationapi// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/noteapi// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/orgapi// <br /> - http://656.qa.docs.sunbird.org/api-documentation/apis/tenantpreferenceapi// <br />  <br/> **Modified sections are -** <br />

**Pull request merged**